pkgs/development/python-modules/outputcheck/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+}:
+
+buildPythonPackage rec {
+  pname = "outputcheck";
+  version = "0.4.2";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "stp";
+    repo = "OutputCheck";
+    rev = "refs/tags/${version}";
+    hash = "sha256-QAlX+dmDKawnEzxtn9TToEmsA9jjzwO04l4bLM3fR/g=";
+  };
+
+  # Fix version number cannot find error
+  prePatch = ''
```

Comment:
prePatch is really only used for operations that need to be done before the patch phase, such as dos2unix, which converts the file format.

Can we move this command to postPatch or preBuild?

Reply:
Tried to move in preBuild and postPatch but `passthru.test` on aarch64-darwin seems to stall